File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/Person.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class Person : DomainResource, IResource
{
    public Base.Address[]? Address { get; set; }
    public Base.ResourceReference? ManagingOrganization { get; set; }
    public Base.HumanName[]? Name { get; set; }
    public string? BirthDate { get; set; }
    public Base.Attachment? Photo { get; set; }
    public Person_Link[]? Link { get; set; }
    public bool? Active { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Base.ContactPoint[]? Telecom { get; set; }
    public string? Gender { get; set; }

    public class Person_Link : BackboneElement
    {
        public required Base.ResourceReference Target { get; set; }
        public string? Assurance { get; set; }
    }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/Person.py
from typing import Optional, List
from base import Address
from base import Attachment
from base import ContactPoint
from base import HumanName
from base import DomainResource
from base import Reference
from base import Identifier
from base import BackboneElement

class Person_Link(BackboneElement):
    target: Reference
    assurance: Optional[str] = None

class Person(DomainResource):
    address: Optional[List[Address]] = None
    managing_organization: Optional[Reference] = None
    name: Optional[List[HumanName]] = None
    birth_date: Optional[str] = None
    photo: Optional[Attachment] = None
    link: Optional[List[Person_Link]] = None
    active: Optional[bool] = None
    identifier: Optional[List[Identifier]] = None
    telecom: Optional[List[ContactPoint]] = None
    gender: Optional[str] = None